src/components/BioCollaboratorList/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import 'node_modules/@biotope/element/lib/host.mixin';
+
+@mixin showSlottedTiles($types...) {
+    @each $type in $types {
+        &.#{$type} {
+            ::slotted(bio-collaborator-tile[type="#{$type}"]) {
+                display: block;
+            }
+        }
+    }
+}
+
+@include host('bio-collaborator-list') {
+	.wrapper {
+		.switch {
+			margin: 20px 0;
```

Comment:
can you add a break between the style for the parent and a nested element, please?